k9mail/src/main/java/com/fsck/k9/activity/MessageCompose.java

```diff
@@ -3022,7 +2995,8 @@ public void onMessageBuildCancel() {
     @Override
     public void onMessageBuildException(MessagingException me) {
         Log.e(K9.LOG_TAG, "Error sending message", me);
-        Toast.makeText(MessageCompose.this, getString(R.string.send_aborted), Toast.LENGTH_LONG).show();
+        Toast.makeText(MessageCompose.this,
+                getString(R.string.send_failed_reason, me.getLocalizedMessage()), Toast.LENGTH_LONG).show();
         currentMessageBuilder = null;
```

Comment:
Please don't display exception messages to the user. We need better error reporting, but that's not what I had in mind :smile: 

Maybe "Send aborted due to an error"?